ARrehab/ARrehab/Minigames/ExpressionGame/ExpressionGame.swift

```diff
@@ -43,7 +48,7 @@ class ExpressionGame: Minigame {
         let defaultConfig = ARWorldTrackingConfiguration()
         self.arSession.run(defaultConfig, options: [])
         self.arSession.delegate = self.viewController.parent as! ARSessionDelegate
-        
+        // TODO return score / completion status.
```

Comment:
todo for once the game design is finalized. also todo for minigame docs to be clearer about what exactly should be returned.

Reply:
Noted